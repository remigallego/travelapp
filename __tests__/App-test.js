/**
 * @format
 */

it('renders correctly', () => {
  expect(1).toBe(1);
  /*   const tree = renderer.create(<App />);
  expect(tree).toMatchSnapshot(); */
});
