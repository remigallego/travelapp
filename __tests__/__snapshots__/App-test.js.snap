// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <RCTScrollView
    contentInsetAdjustmentBehavior="automatic"
    style={Object {}}
  >
    <View>
      <Text
        style={
          Object {
            "fontSize": 20,
          }
        }
      >
        Hello
      </Text>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
